### ISSUE TITLE
React gracefully on LDAP timeouts

### ISSUE THREAD

**Issue by LukasReschke** [nextcloud/server#7091]
When the LDAP server isn't reacting we should react gracefully and not make the requests to the server timeout. – Maybe show the maintenance page or so?

**Comment by schiessle** [comment]
yes, maybe even some error message like "user back-end not available" or something similar.

**Comment by blizzz** [comment]
I am not sure i understand what you mean in the ticket. If LDAP does not react, it will throw ServerNotAvailable. Should we have a shorter timeout for the connect to it? Or you only want to have a nicer error message? Or something else?

**Comment by LukasReschke** [comment]
So the problem on c.nc.c was that the site simply didn't load anymore and loading timed out completely and you only saw the browser error page. But no indication of any error except "Couldn't load page" :)

**Comment by blizzz** [comment]
Ah, didn't know this. This makes sense of course

**Comment by szaimen** [comment]
I suppose this issue is still valid? If not, please close this issue!
